packages/cli/internal/pkg/cli/workflow/workflow_tasks.go

```diff
@@ -32,6 +38,23 @@ func (m *Manager) GetWorkflowTasks(runId string) ([]Task, error) {
 
 }
 
+func (m *Manager) GetRunLog(runId string) (RunLog, error) {
+
```

Comment:
for a manger's method it is a must to check error first thing, and exit if it's not `nil`.